android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/web/WebSocialSession.kt

```diff
@@ -0,0 +1,21 @@
+package io.logto.sdk.android.auth.social.web
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import io.logto.sdk.android.completion.Completion
+
+class WebSocialSession(
```

Comment:
should we define a protocol for social sessions like Swift SDK?

Reply:
LOG-2125

Reply:
done.

Reply:
where's the protocol?

Reply:
I misunderstand the `protocol` you mentioned. I thought we were talking about the convention about the UI & the native side.
Updated, PTAL.

Reply:
We have a `SocialSession` interface. this `WebSocialSession` implements the interface.